commit.go

```diff
@@ -47,22 +48,22 @@ func (b *Builder) Commit(dest types.ImageReference, options CommitOptions) error
 	}
 	src, err := b.makeContainerImageRef(options.Compression)
 	if err != nil {
-		return err
+		return fmt.Errorf("error recomputing layer digests and building metadata: %v", err)
 	}
 	err = copy.Image(policyContext, dest, src, getCopyOptions(options.ReportWriter))
 	if err != nil {
-		return err
+		return fmt.Errorf("error copying layers and metadata: %v", err)
 	}
 	if len(options.AdditionalTags) > 0 {
 		switch dest.Transport().Name() {
 		case storage.Transport.Name():
 			img, err := storage.Transport.GetStoreImage(b.store, dest)
 			if err != nil {
-				return err
+				return fmt.Errorf("error locating just-written image: %v", err)
```

Comment:
Could you add the value from b.store to the message to let the user know what it was looking at in particular when there was an error?  Or do you think that wouldn't help the debugging process later on down the road?